tutorials/reduction_sequence_science.ipynb

```diff
@@ -147,12 +112,16 @@
    "outputs": [],
    "source": [
     "sci_rss = []\n",
+    "aaomega_arms = {'blue': 1, 'red': 2}\n",
+    "# Choose which arm of the spectrograph is going to be used\n",
+    "arm = 'blue'\n",
+    "path_to_data = 'data'\n",
+    "\n",
     "adr_set = []\n",
     "\n",
     "#for i in [31, 32, 33]:\n",
     "for i in [34, 35, 36]:\n",
-    "    filename = os.path.join(\"data\", \"koala\", grating,\n",
-    "                            f\"27feb{aaomega_gratings[grating]}00{i}red.fits\")\n",
+    "    filename = f\"data/koala/385R/27feb200{i}red.fits\"\n",
     "    rss = koala_rss(filename)\n",
```

Comment:
I believe making use of the `os` library for dealing with the paths is always much better than hardcoding the paths manually, why did you change that?